content/_data/sidebar.json

```diff
@@ -832,6 +832,10 @@
               "title": "dom",
               "slug": "dom"
             },
+            {
```

Comment:
Can you remove this change here? I made a separate PR here https://github.com/cypress-io/cypress-documentation/pull/4303